tests/parser-test-files/parse-if.txt

```diff
@@ -0,0 +1,63 @@
+================================================================================
+Malformed if expression
+================================================================================
+
+(package test-package)
+
+(define x (if))
+
+--------------------------------------------------------------------------------
+
+error: Malformed if expression
+  --> test:3:14
+   |
+ 3 |  (define x (if))
+   |               ^ expected a predicate
+
+================================================================================
+Malformed if expression
+================================================================================
+
+(package test-package)
+
+(define x (if 1))
```

Comment:
All of these `if` cases, even though they fail because of syntax, would also fail because they aren't using `Boolean` predicates. I'd swap out the first parameters for `True`/`False`s:
```
(define x (if True))

(define x (if True 2))

(define x (if True 2 3 4))
```

This isn't important for tests as much as it is important for documentation.